src/test/java/com/fishercoder/_606Test.java

```diff
@@ -38,4 +38,12 @@ public void test2(){
         System.out.println("Test2");
         assertEquals("1(2()(4))(3)", test.tree2str(t));
     }
+    @Test
```

Comment:
So, each time when I PR for a file and I need to close it for the second file that I want to PR?